processing/data_collection/gazette/spiders/sp_itapirapua_paulista.py

```diff
@@ -0,0 +1,8 @@
+from gazette.spiders.instar_base import BaseInstarSpider
+
+
+class SpItapirapuaPaulistaSpider(BaseInstarSpider):
+    TERRITORY_ID = "3522653"
+    name = "sp_itapira_puapaulista"
```

Comment:
Acho que aqui o nome da cidade ficou quebrado no lugar errado uehehueh
```suggestion
    name = "sp_itapirapua_paulista"
```

Reply:
Malz rsrs fiz a correção aqui